src/styles/components/_componentCard.scss

```diff
@@ -13,6 +13,8 @@
 
```

Comment:
What if under `.component-card` you also added `width: 100%`? It would be on line 8.

Reply:
I didn't notice a change, but it didn't hurt either!